e2e/mocks/data-api/person.json

```
@@ -17,7 +17,7 @@
   "PriorUWNetIDs": [],
   "PriorUWRegIDs": [],
   "RegisteredFirstMiddleName": "Jane __RegId__",
```

`RegisteredFirstMiddleName` should match `RegisteredName`
```suggestion
  "RegisteredFirstMiddleName": "Janeth __RegId__",
```